packages/web3-eth-accounts/src/account.ts

```diff
@@ -0,0 +1,70 @@
+import { utils, getPublicKey } from 'ethereum-cryptography/secp256k1';
+import { toChecksumAddress, bytesToHex, sha3Raw, HexString } from 'web3-utils';
+import { InvalidPrivateKeyError, PrivateKeyLengthError } from './errors';
+
+// Will be added later
+export const encrypt = (): boolean => true;
+
+// Will be added later
+export const sign = (): boolean => true;
+
+// Will be added later
+export const signTransaction = (): boolean => true;
+
+/**
+ * Get account from private key
+ */
+export const privateKeyToAccount = (
+	privateKey: string | Buffer,
+): {
+	address: string;
+	privateKey: string;
+	signTransaction: () => boolean; // From 1.x
+	sign: () => boolean;
+	encrypt: () => boolean;
+} => {
+	if (!privateKey) {
+		throw new InvalidPrivateKeyError(privateKey);
+	}
+
+	const stringPrivateKey =
+		typeof privateKey === 'object' ? Buffer.from(privateKey).toString('hex') : privateKey;
+
+	const updatedKey = stringPrivateKey.startsWith('0x')
+		? stringPrivateKey.slice(2)
+		: stringPrivateKey;
+
+	// Must be 64 hex characters
```

Comment:
```suggestion
    // TODO Replace with isHexString32Bytes function in web3-eth PR:
    // https://github.com/ChainSafe/web3.js/blob/afa6a4bcdfbc122246f35e749d251fd1afb683d0/packages/web3-utils/src/validation.ts#L437
	// Must be 64 hex characters
```

Reply:
Could also use the method [validateHexString32Bytes](https://github.com/ChainSafe/web3.js/pull/4525/files#diff-170228df6bd950804f5f0071be007d1f2acba8a41ef2f9959b815e6ca82a1d97R440) which will throw a `InvalidHexStringError` with the message: `not a valid 32 byte hex string` - Unless you'd prefer the error be more specific with `PrivateKeyLengthError` then you could use [isHexString32Bytes](https://github.com/ChainSafe/web3.js/blob/afa6a4bcdfbc122246f35e749d251fd1afb683d0/packages/web3-utils/src/validation.ts#L437) which returns a boolean

Reply:
Noted, will be added when web3-eth PR is merged